plugins/README.md

```diff
@@ -0,0 +1,51 @@
+### Oh My Bash: Enhancing Your Development Experience
```

Comment:
I'm wondering about the heading level. Why doesn't this start with the `<h1>` level (i.e. single `#`)? Could you make it `<h1>`?